code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -158,9 +156,54 @@
 
 /obj/item/reagent_containers/glass/beaker/jar
 	name = "honey jar"
-	desc = "A jar for honey. It can hold up to 60 units of sweet delight. Unable to withstand reagents of an extreme pH."
-	icon = 'icons/obj/chemical.dmi'
-	icon_state = "vapour"
+	desc = "A jar for honey. It can hold up to 50 units of sweet delight. Unable to withstand reagents of an extreme pH."
+	icon_state = "honey"
+
+/obj/item/reagent_containers/glass/beaker/glass_dish
+	name = "glass dish"
+	desc = "A tiny glass dish. It can hold up to 3 units. Unable to withstand reagents of an extreme pH."
+	materials = list(MAT_GLASS=500)
+	icon_state = "glass_disk"
+	possible_transfer_amounts = list(0.1,0.5,0.75,1,2,3)
+	volume = 3
+
+/obj/item/reagent_containers/glass/beaker/glass_dish/Initialize()
+	beaker_weakness_bitflag &= ~PH_WEAK
+	. = ..()

```

Comment:
Just add `beaker_weakness_bitflag  = NONE` to the typepath vars and delete these lines.

Reply:
These lines are needed so that they beakers understand their volume on creation

Reply:
This is the first thing done on these beakers' `Initialize()` before the parent call,  why would it even differ from having the bitflag defined on the typepath? Please keep it consistent with all the other datums that override initial vars on their path instead of doing so on `Initialize()`.
Otherwise explain yourself, cause I can't even catch a glimpse of that comment.

Reply:
without that line when made they have a volume of 1
also they are weak to PH so the flag is needed.

Reply:
That's because you unawarely wrote down `volume = NONE` instead of `beaker_weakness_bitflag = NONE`. Please be more careful next time and do what I said.